packages/fxa-content-server/app/tests/spec/views/base.js

```diff
@@ -848,75 +847,6 @@ describe('views/base', function () {
     });
   });
 
-  describe('BaseView.preventDefaultThen', function () {
```

Comment:
I don't see these tests added anywhere else, can you create a test new module for each of the methods?